src/scss/_variables.scss

```diff
@@ -3,4 +3,4 @@ $o-colors-is-silent: true !default;
 $o-colors-palette: () !default;
 $o-colors-usecases: () !default;
 
-$o-colors-brand: if(global-variable-exists(o-brand), $o-brand, 'master');
+$_o-colors-brand: if(global-variable-exists(o-brand), $o-brand, 'master');
```

Comment:
`$o-brand` might exist and be `null`, so that case should be handled too.

Reply:
🤦‍♀️